### PR TITLE
address comments, revert test input to error causing

### PR DESCRIPTION
**Pull request by nb2499**

### CODE REVIEW

**Comment by nb2499** [comment]
address comments, revert test input into error throwing

File: assignment/phoenix/test.m [hunk 1]
```diff
@@ -13,7 +13,7 @@ function test()
 
 % check for valid URL
 try
-	randomURL = "http://cidlab.com/files/cogs205b.csv";
+	randomURL = "http://cdlab.com/files/cogs205b.csv";
 	getData(randomURL);
 	errorThrown = true;
```
Comment (joachimvandekerckhove) [inline assignment/phoenix/test.m:18]: The idea is right but the logic has an error here.  The try-block on the line where an error is thrown, so if all goes well, line 18 is never reached (but line 20 is instead).  So for a _bad_ input URL (causing an error), errorThrown will be **false**.

Comment (joachimvandekerckhove) [reply assignment/phoenix/test.m:18]: (Same issue in a few tests.)

File: assignment/phoenix/test.m [hunk 2]
```diff
@@ -23,75 +23,73 @@ function test()
...
 
-
-%% Negative Tests
 % Valid Data: NaNs
 try
-	d = [1 2 0.4 4 7; 4 6 6 4 3];
+	d = [1 2 0.4 NaN 7; 4 6 6 4 3];
 	phoenix.Norm2d.estimate(d);
-	errorThrown = true;
+	errorNotThrown = true;
```
Comment (joachimvandekerckhove) [inline assignment/phoenix/test.m:38]: This is how it's supposed to be!

Comment (joachimvandekerckhove) [inline assignment/phoenix/test.m:38]: This variable is named correctly :+1:

File: assignment/phoenix/test.m [hunk 3]
```diff
@@ -23,75 +23,73 @@ function test()
...
 catch
-	errorThrown = false;
+	errorNotThrown = false;
 end
-assertErrorThrown(errorThrown, "Data: no NaNs")
+assertErrorThrown(errorNotThrown, "Data: no NaNs")
```
Comment (joachimvandekerckhove) [inline assignment/phoenix/test.m:42]: ... but this is not what you mean, I think.  Here, the test suite will _fail_ if an error is thrown during the test.  But you (I assume) want an error to be thrown if there are NaNs in the data, so the test should in fact fail if an error was **not** thrown.

**Review by joachimvandekerckhove — APPROVED** [review]

**Pull request merged**